### PR TITLE
Updated Scratch recipe

### PR DESCRIPTION
**Pull request by corrionc**
They changed the download filename slightly.

### CODE REVIEW

File: Disabled/Scratch.xml [hunk 1]
```diff
@@ -6,12 +6,12 @@
 		<AutoInstall>True</AutoInstall>
 		<UserDocumentation>https://scratch.mit.edu/</UserDocumentation>
 		<Icon>Scratch.png</Icon>
-		<FolderPath></FolderPath>
+		<FolderPath>Costa</FolderPath>
```
Comment (asjimene) [inline Disabled/Scratch.xml:9]: Not sure this part is needed?

**Comment by corrionc** [comment]
Good catch.  I _think_ the changes requested feature took care of the request, but if not I've updated the file.

**Comment by asjimene** [comment]
Yep the changes actually make a commit!

**Comment by asjimene** [comment]
Thank you!

**Pull request merged**